Add confirmation page for workshops (#2367)
pulumi/docs: content/confirmation/_index.md

---
private: true
title: Confirmation

> meta_desc: This page is provides confirmation about register for a Pulumi event, webinar, or workshop.

Reviewer: ```suggestion
meta_desc: This page provides confirmation after registering for a Pulumi event, webinar, or workshop.
```

block_external_search_index: true

> ---

Reviewer: Do we ever expect to use this index page? Instead of having it as effectively empty page, we could consider just redirecting to the homepage, e.g. changing the content of this file to:

```markdown
---
redirect_to: /
---
```

Author: Yup, that worked. I removed all the extra stuff and just have the redirect now.
